pages/1.12/installing/index.md

```diff
@@ -11,9 +11,11 @@ The installation of DC/OS involves configuring your infrastructure, and installi
 
 The DC/OS installation methods are as follows:
 
-- **Mesosphere supported installation:** These methods are used for trials and proofs of concept (PoC) only. To test or demo DC/OS on Azure, AWS, or GCP, follow the [Mesosphere supported installation methods](#mesosphere-supported).
+- **Mesosphere-supported installation:** Use the [Mesosphere-supported installation methods](#mesosphere-supported) to test or demo DC/OS on AWS, Azure, or GCP using an Universal Installer.
```

Comment:
Shouldn't this be "Mesosphere Supported Installation"?  no hyphens, capitalize

Reply:
This is “Mesosphere-supported” and “Community-supported” (as they are compound adjectives).